main.go

```diff
@@ -678,6 +678,9 @@ func handleTriggerUninstall() {
 	triggerUninstallFlagSet.StringVar(
 		&policyNamespace, "policy-namespace", "", "The namespace of where ConfigurationPolicy objects are stored",
 	)
+	triggerUninstallFlagSet.StringVar(
+		&additionalNamespace, "additional-namespace", "", "An additional namespace for ConfigurationPolicy objects",
```

Comment:
Should config-controller watch this namespace too?

Reply:
This `trigger-uninstall` flag is meant to match when the main controller is watching "open-cluster-management-policies": https://github.com/open-cluster-management-io/config-policy-controller/blob/main/main.go#L259

Reply:
I'll need to add it in the addon-controller